shared/domains/chelonia/db.js

```diff
@@ -17,6 +17,7 @@ const getLogHead = (contractID: string): string => `${headPrefix}${contractID}`
 type HEADInfo = { HEAD: string; height: number }
 
 export const checkKey = (key: string): void => {
+  if (key.startsWith('private/')) return
```

Comment:
Is it also still allowing private keys to be read from the database by anyone using the `/file` API..?